assets/src/blocks/Happypoint/HappypointFrontend.js

```diff
@@ -42,19 +48,25 @@ export const HappypointFrontend = ({
       </picture>
       <div className="container">
         <div className="row justify-content-md-center">
-          {mailing_list_iframe && url && (
-            <div className="col-md-10 happy-point" id="happy-point" data-src={url}>
-              <iframe
-                src={url}
-                cellSpacing={0}
-                allowtransparency="true"
-                frameBorder={0}
-                scrolling="no"
-                width="100%"
-                loading="lazy"
-              />
-            </div>
-          )}
+          {use_embed_code
+            ? <div className="col-md-10" id="happy-point" />
```

Comment:
Can you also add an `mt-5` class here? Just to make the iframe have some spacing from the top.